pkg/sql/sqlbase/structured.proto

```diff
@@ -621,6 +621,7 @@ message TableDescriptor {
     // Start value of the sequence.
     optional int64 start = 4 [(gogoproto.nullable) = false];
     // Whether to wrap around when the min or max value is hit.
+    // Currently unimplemented (doesn't parse, value ignored).
```

Comment:
Any reason to keep this field around? If you decide to keep it around I'd mark yourself with a `TODO(vilterp)` so whoever comes across it knows who to ask for context.